tests/events/18-suite-event-mail.t

```diff
@@ -16,6 +16,7 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #-------------------------------------------------------------------------------
 # Test event mail.
+CYLC_TEST_IS_GENERIC=false
```

Comment:
It is configured by line 32 in this file. In `cylc.scheduler.Scheduler.run_event_handlers`, the value of this setting should be used as the value of the environment variable `smtp` on launching the `mail` command.